example/ios/Podfile.lock

```diff
@@ -13,7 +13,9 @@ PODS:
     - Flutter
   - fluttertoast (0.0.2):
     - Flutter
+    - Toast
```

Comment:
This was generated as a part of release process while running `pod repo update` ( because of error running following ) and then ran following
`pod install --project-directory=example/ios` and `pod install --project-directory=test_integration/ios`